cluster/saltbase/salt/mount-propagation/init.sls

```diff
@@ -0,0 +1,15 @@
+{% if not pillar.get('is_systemd') %}
```

Comment:
Is this restrictive enough to just select debian wheezy (which according to the PR comment is the only place this should run)?